lib/common/MongoDbLogger.ts

```diff
@@ -0,0 +1,31 @@
+import { LoggerState, MongoClient } from 'mongodb';
+import Logger from '../common/Logger';
+
+/**
+ * Base class that contains the common logging functions for mongodb.
+ */
+export default class MongoDbLogger {
+  static setCommandLogger (client: MongoClient) {
+    client.on('commandSucceeded', (event: any) => {
```

Comment:
For my learning: why do you need to do both explicit event logging AND set custom logger? Is it because the custom logger doesn't actually log these events even though you've set it on `debug` level? If so, what does custom log? Do you have an example for either cases?